cogs/welcome/welcome.py

```diff
@@ -97,14 +99,45 @@ async def sendWelcomeMessageChannel(self, newUser: discord.Member, guild: discor
             message += splitMessage[x]
             if x < len(splitMessage) - 1:
                 message += newUser.mention
-        fullMessage = f"""{message} Welcome to SFU Anime Club's Discord!
 
-Please check out <#225044755887685632> for all of our basic information and rules, and feel free to introduce yourself with the pinned template in this channel!
-
-We hope you enjoy your stay~
+        joinLogEnabled = await self.config.guild(newUser.guild).logJoinEnabled()
+        joinLogChannel = await self.config.guild(newUser.guild).logJoinChannel()
+        errorChannel = discord.utils.get(
+            newUser.guild.text_channels, id=joinLogChannel
+        )
+        try:
+            await channel.send(message)
+        except (discord.Forbidden, discord.HTTPException) as errorMsg:
+            LOGGER.error(
+                "Could not send message, the user may have"
+                "turned off DM's from this server."
+                " Also, make sure the server has a title "
+                "and message set!",
+                exc_info=True,
+            )
+            LOGGER.error(errorMsg)
+            if joinLogEnabled and errorChannel:
+                await errorChannel.send(
+                    ":bangbang: ``Server Welcome:`` User "
+                    f"{newUser.name}#{newUser.discriminator} "
+                    f"({newUser.id}) has joined. Could not send "
+                    "DM!"
+                )
+                await errorChannel.send(errorMsg)
+        else:
+            if joinLogEnabled and errorChannel:
+                await errorChannel.send(
+                    f":o: ``Server Welcome:`` User {newUser.name}#"
+                    f"{newUser.discriminator} ({newUser.id}) has "
+                    "joined. DM sent."
+                )
+                LOGGER.info(
+                    "User %s#%s (%s) has joined.  DM sent.",
```

Comment:
can adjust log message to something like:
```suggestion
                "User %s#%s (%s) has joined. Posted welcome message.",
```

Reply:
Done